src/screens/resources/resource.js

```diff
@@ -0,0 +1,46 @@
+import { cx } from "../../utils/cx";
+
+export function Resource({
+  thumbnail,
+  title,
+  subtext = "",
+  children,
+  links,
+  containThumbnail = false,
+}) {
+  return (
+    <div className="flex w-9/10 flex-shrink-0 snap-center flex-col overflow-hidden rounded-2xl border border-white bg-white/50 pb-4 lg:w-auto lg:flex-row lg:pb-0">
+      <div className="grid place-items-center bg-resource bg-cover">
+        <img
+          src={`/images/other-resources/${thumbnail}`}
+          alt=""
+          className={cx(
+            "aspect-square h-full w-full md:aspect-video lg:aspect-square lg:w-56",
+            containThumbnail ? "object-contain lg:h-4/5" : "object-cover"
+          )}
+        />
+      </div>
+      <div className="flex flex-1 flex-col gap-1 overflow-hidden p-4 lg:p-6">
+        <h3 className="text-xl font-bold line-clamp-2 lg:text-2xl lg:line-clamp-none">
+          {title}
+        </h3>
+        <span>{subtext}</span>
+
+        <p className="line-clamp-3 lg:line-clamp-none">{children}</p>
+      </div>
+      <div className="flex flex-col gap-4 px-4 lg:w-56 lg:py-6 lg:pr-6">
+        {links.map(([text, url], i) => (
+          <a
+            key={i}
```

Comment:
Index as a key is an anti-pattern and should only be used as a last resort (explained [here](https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/)). Using the `url` would be better here:
```suggestion
        {links.map(([text, url]) => (
          <a
            key={url}
```

Reply:
I usually go with the array index if it's a static list, as the article says is okay:

> To help you decide, I put together three conditions which these examples have in common:
> 
> 1. the list and items are static–they are not computed and do not change;
> 2. the items in the list have no ids;
> 3. the list is never reordered or filtered.
>  
> When all of them are met, you may safely use the index as a key.

But I've changed it anyway!

Reply:
The [React docs](https://reactjs.org/docs/lists-and-keys.html#keys) say:
>When you don’t have stable IDs for rendered items, you may use the item index as a key as a last resort

That's why, if something unique is available, I always use that. I'm being very nitpicky here, though!